pr_agent/settings/configuration.toml

```diff
@@ -33,10 +33,30 @@ add_original_user_description=false
 keep_original_user_title=false
 use_bullet_points=true
 extra_instructions = ""
+
 # markers
 use_description_markers=false
 include_generated_by_header=true
 
+[custom_labels."Bug fix"]
```

Comment:
this takes a lot of room in the main configuration file

should move to a separate file

Reply:
No
move that big chunk of text to another file.
its unreasonable to dedicated so many lines in a general configuration to a tailor made optional feature.